src/containers/Router/Routes/PrivateRoute/PrivateRoute.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from "react";
+import { Navigate } from "react-router-dom";
+
+import { IRoutesProps } from "../types/Routes.type";
+import { useAppSelector } from "@/src/redux/hooks";
+import { getTokenFromLC, links } from "@/src/utils";
+
+const PrivateRoute: FC<IRoutesProps> = ({
+  component: Component,
+  redirectUrl = links.LOG_IN,
+}) => {
+  const { isAuth } = useAppSelector((state) => state.userSlice);
+  if (!getTokenFromLC()) return <Navigate to="/" replace />;
```

Comment:
The access token can be outdated while there might be a valid refresh token. In the case the access token is invalid, we should try to refresh it. When the fetchIsAuthUser is executed, the axios interceptors will try to pass an access token into Authorization header and if any problems occur, they will try to refresh tokens and the user will be authorized. That is why we should check if isAuth is true. But I think you might have the reasons to define it so and I might have overlooked something